src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+}
+
+.logo {
+  margin: 10px 0 10px 50px;
```

Comment:
1. Better to add left/right padding for parent header element instead of adding left/right margins separately to the logo and navigation, shoud be smth like this:
![image](https://github.com/mate-academy/layout_moyo-header/assets/54936044/3d42d22e-8f95-4672-9908-a5bfbfec6b88)
2. Instead of centering the logo with padding, please add a rule to the parent flex header element that centers all children on the y-axis